Makefile

```diff
@@ -7,7 +7,7 @@ all: help
 
 .PHONY: tidy
 tidy: ## Updates the go modules
-	go mod tidy -compat=1.17
+	go mod tidy
```

Comment:
Do we need `-compat=1.18`?

Reply:
yes. I found that the dapr respository was golang1.18.